GOVERNANCE.md

```diff
@@ -11,21 +11,36 @@ The Fluentd community adheres to the following principles:
 
 ## Voting
 
-The Fluentd project employs "organization voting" to ensure no single organization can dominate the project.
+The Fluentd project employs "organization voting" and "Vetoes" to ensure no single organization can dominate the project.
+
+For formal votes, a specific statement of what is being voted on should be added to the relevant GitHub issue or PR. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+
+All the following conditions must be satisfied to approve the formal votes.
+
+- No effective objection ballot: See Vetoes, below.
+- At least effective __2-organization__ affirmative vote: See Organization Vote, below.
+- At least effective __3-maintainers__ affirmative vote.
+- At least __2-week__ for voting.
+
+### Organization Vote
 
 Individuals not associated with or employed by a company or organization are allowed one organization vote. Each company or organization (regardless of the number of maintainers associated with or employed by that company/organization) receives one organization vote.
 
 In other words, if two maintainers are employed by Company X, two by Company Y, two by Company Z, and one maintainer is an un-affiliated individual, a total of four "organization votes" are possible; one for X, one for Y, one for Z, and one for the un-affiliated individual.
 
 Any maintainer from an organization may cast the vote for that organization.
 
-For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR, and a link to that issue or PR added to the maintainers meeting agenda document. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
+### Vetoes
+
+The proposal is not approved as long as any maintainer votes an effective objection ballot.
+
+The maintainer who votes an objection ballot must explain the reason for the objection. Without reasonable justification, the objection ballot is not considered effective.
```

Comment:
It may be too sensitive, I have a concern about who judge "reasonable justification"?
It depends on which side you stand.

Can we make an agreement about "what is reasonable justification"?

Reply:
Thanks!

There is some concern about not stating anything in this regard, but I think it would be difficult to set rules in too much detail for this.

I'm referring to Apache Voting Process here as well.

* https://www.apache.org/foundation/voting#Veto

Reply:
Hmm, then how about phrasing **reasonable "technical" justification** ?

Reply:
I am concerned about whether it is appropriate to limit it to "technical" matters since I think that votes could be taken on non-technical matters as well.

For example, in a vote on a rule change such as this one, technical knowledge is not relevant.

Reply:
Ah, that's right.